### DISCUSSION TITLE
Click to copy classnames in docs

### DISCUSSION THREAD

**Discussion by givvemee** [tailwindlabs/tailwindcss#13146] [Ideas]
I spend a lot of time to style my project with tailwind these days, so docs is always by my side. 
However, I don't feel good sometimes I should drag and cmd+c classnames then cmd+v in my code. 
How about making a feature when the user click Class (left side of table in the pic) and then copy it to clipboard? 

<img width="492" alt="Screenshot 2024-03-08 at 1 11 26 PM" src="https://github.com/tailwindlabs/tailwindcss/assets/89691274/90f0f5c9-12a4-4e38-a420-b1b5eda2fe81">

**Comment by fliklab** [comment]
It would be useful if I could copy those classnames with just a click.
